after gcc linking has changed, all the libraries must be explicitely specified 
This patch avoids these linking errors:

| make[1]: Entering directory `/disk0/pokybuild/build1/tmp/work/i586-poky-linux/matchbox-stroke-0.0+svnr1820-r0/matchbox-stroke'^M
| Making all in src^M
| make[2]: Entering directory `/disk0/pokybuild/build1/tmp/work/i586-poky-linux/matchbox-stroke-0.0+svnr1820-r0/matchbox-stroke/src'^M
| ccache i586-poky-linux-gcc -march=i586 --sysroot=/disk0/pokybuild/build1/tmp/sysroots/i586-poky-linux  -fexpensive-optimizations -fomit-frame-pointer -frename-registers -O2 -ggdb -feliminate-unused-debug-types  -Wl,-O1 -Wl,--hash-style=gnu -Wl,--as-needed -o matchbox-stroke matchbox-stroke.o matchbox-stroke-ui.o matchbox-stroke-recog.o matchbox-stroke-mode.o matchbox-stroke-action.o config-parser.o util-hash.o util.o -lXft -lX11 -lXtst -lfakekey   -lexpat -lm^M
| /disk0/pokybuild/build1/tmp/sysroots/x86_64-linux/usr/libexec/i586-poky-linux/gcc/i586-poky-linux/4.5.1/ld: u: invalid DSO for symbol `XRenderFindVisualFormat' definition^M
| /disk0/pokybuild/build1/tmp/sysroots/i586-poky-linux/usr/lib/libXrender.so.1: could not read symbols: Bad value^M
| collect2: ld returned 1 exit status^M
| make[2]: *** [matchbox-stroke] Error 1

Nitin A Kamble <nitin.a.kamble@intel.com>
Date: 2011/01/11

Upstream-Status: Pending

Index: matchbox-stroke/configure.ac
===================================================================
--- matchbox-stroke.orig/configure.ac
+++ matchbox-stroke/configure.ac
@@ -38,7 +38,7 @@ AC_ARG_WITH(expat-lib,         
 	   expat_lib=$withval, expat_lib=yes)
 
 
-PKG_CHECK_MODULES(MBSTROKE, xft libfakekey,,
+PKG_CHECK_MODULES(MBSTROKE, xft libfakekey xrender,,
 	         AC_MSG_ERROR([*** Required Librarys not found ***]))
 
 dnl ------ Expat ------------------------------------------------------------
@@ -160,4 +160,4 @@ echo "
         compiler:               ${CC} 
 
         Building with Debug:    ${enable_debug}
-"
\ No newline at end of file
+"
